std/UInt.hx

```diff
@@ -31,76 +30,107 @@
 @:notNull
 @:runtimeValue
 @:analyzer(no_const_propagation)
-abstract UInt to Int from Int
-{
-	@:commutative @:op(A+B) private static function addI(lhs:UInt, rhs:Int):UInt;
-	@:commutative @:op(A+B) private static function addF(lhs:UInt, rhs:Float):Float;
-	@:op(A+B) private static function add(lhs:UInt, rhs:UInt):UInt;
-
-	@:commutative @:op(A*B) private static function mulI(lhs:UInt, rhs:Int):UInt;
-	@:commutative @:op(A*B) private static function mulF(lhs:UInt, rhs:Float):Float;
-	@:op(A*B) private static function mul(lhs:UInt, rhs:UInt):UInt;
-
-	@:op(A%B) private static function modI(lhs:UInt, rhs:Int):UInt;
-	@:op(A%B) private static function modF(lhs:UInt, rhs:Float):Float;
-	@:op(A%B) private static function mod(lhs:UInt, rhs:UInt):UInt;
-
-	@:op(A-B) private static function subI(lhs:UInt, rhs:Int):UInt;
-	@:op(A-B) private static function subF(lhs:UInt, rhs:Float):Float;
-	@:op(A-B) private static function sub(lhs:UInt, rhs:UInt):UInt;
-
-	@:op(A/B) private static function divI(lhs:UInt, rhs:Int):Float;
-	@:op(A/B) private static function divF(lhs:UInt, rhs:Float):Float;
-	@:op(A/B) private static function div(lhs:UInt, rhs:UInt):Float;
-
-	@:commutative @:op(A|B) private static function orI(lhs:UInt, rhs:Int):UInt;
-	@:op(A|B) private static function or(lhs:UInt, rhs:UInt):UInt;
-
-	@:commutative @:op(A^B) private static function xorI(lhs:UInt, rhs:Int):UInt;
-	@:op(A^B) private static function xor(lhs:UInt, rhs:UInt):UInt;
-
-	@:commutative @:op(A&B) private static function andI(lhs:UInt, rhs:Int):UInt;
-	@:op(A&B) private static function and(lhs:UInt, rhs:UInt):UInt;
-
-	@:op(A<<B) private static function shl(lhs:UInt, rhs:Int):UInt;
-	@:op(A>>B) private static inline function shr(lhs:UInt, rhs:Int):UInt return lhs >>> rhs;
-	@:op(A>>>B) private static function ushr(lhs:UInt, rhs:Int):UInt;
-
-	@:op(A>B) private static function gt(lhs:UInt, rhs:UInt):Bool;
-	@:op(A>=B) private static function gte(lhs:UInt, rhs:UInt):Bool;
-	@:op(A<B) private static function lt(lhs:UInt, rhs:UInt):Bool;
-	@:op(A<=B) private static function lte(lhs:UInt, rhs:UInt):Bool;
-
-	@:op(A>B) private static function gtf(lhs:UInt, rhs:Float):Bool;
-	@:op(A>B) private static function gtf2(lhs:Float, rhs:UInt):Bool;
-	@:op(A>=B) private static function gtef(lhs:UInt, rhs:Float):Bool;
-	@:op(A>=B) private static function gtef2(lhs:Float, rhs:UInt):Bool;
-	@:op(A<B) private static function ltf(lhs:UInt, rhs:Float):Bool;
-	@:op(A<B) private static function ltf2(lhs:Float, rhs:UInt):Bool;
-	@:op(A<=B) private static function ltef(lhs:UInt, rhs:Float):Bool;
-	@:op(A<=B) private static function ltef2(lhs:Float, rhs:UInt):Bool;
+abstract UInt to Int from Int {
+	@:commutative @:op(A + B) private static function addI(lhs:UInt, rhs:Int):UInt;
+
```

Comment:
No newlines when there is consecutive methods with no body seems like a good rule

Reply:
Agreed. That is normally the logic for externs / interfaces / etc, I think the formatter doesn't account for body-less functions being a thing in externs too yet.